tools/pythonpkg/src/pyconnection.cpp

```diff
@@ -75,6 +75,8 @@ void DuckDBPyConnection::Initialize(py::handle &m) {
 	    .def("view", &DuckDBPyConnection::View, "Create a relation object for the name'd view", py::arg("view_name"))
 	    .def("values", &DuckDBPyConnection::Values, "Create a relation object from the passed values",
 	         py::arg("values"))
+	    .def("value_rows", &DuckDBPyConnection::ValuesRows, "Create a relation object from the passed values (as a rowset)",
+	         py::arg("values"))
```

Comment:
This argument would probably be better named `rows`?

Reply:
I've updated it to such in the latest commit